{{cookiecutter.repo_name}}/config/settings/production.py

```diff
@@ -118,7 +118,7 @@
 
 # URL that handles the media served from MEDIA_ROOT, used for managing
 # stored files.
-{% if cookiecutter.use_whitenoise == 'y' -%}
+{% if cookiecutter.use_whitenoise == "y" -%}
```

Comment:
My main contention is that the cookiecutter templates are used throughout the codebase in other non-Python files. For instance in a JSON file it would be consistent on a file level to use double quotes everywhere instead. 

I guess every file in this repo is a cookiecutter template first and another file type second. Like there can be a consistency 'boundary' between the two file types where as a template, double quotes make sense, and as a Python file single quotes make sense. 

Maybe I'm thinking about this too much. 😄 I'd like to get this merged though. Do you want me to just make all quotes in Python files single or keep the template statements double quoted?